.clang-tidy

```diff
@@ -0,0 +1,2 @@
+---
+Checks: '-*,modernize-deprecated-headers,modernize-redundant-void-arg,modernize-replace-random-shuffle,modernize-use-equals-default,modernize-use-equals-delete,modernize-use-nullptr,modernize-use-override,modernize-use-using'
```

Comment:
Would it be possible to have them one per line? If not, then please ignore the comment

Reply:
Not as far as I know